### PR TITLE
SAK-44582 SIGNUP - Signup isn't available for students

### PR DESCRIPTION
**Pull request by josecebe**
https://jira.sakaiproject.org/browse/SAK-44582

### CODE REVIEW

**Review by bjones86 — APPROVED** [review]
How did this change from being a zero based index to a 1 based index?

File: signup/tool/src/java/org/sakaiproject/signup/tool/jsf/NewSignupMeetingBean.java [hunk 1]
```diff
@@ -170,7 +170,7 @@ public NewSignupMeetingBean() {
     }
 
     public List<SelectItem> getSlots() {
-        slots = IntStream.range(0, numberOfSlots).mapToObj(i -> new SelectItem(i, String.valueOf(i))).collect(Collectors.toList());
+        slots = IntStream.range(1, numberOfSlots+1).mapToObj(i -> new SelectItem(i, String.valueOf(i))).collect(Collectors.toList());
```
Comment (ern) [inline signup/tool/src/java/org/sakaiproject/signup/tool/jsf/NewSignupMeetingBean.java:173]:
```suggestion
        slots = IntStream.range(1, numberOfSlots + 1).mapToObj(i -> new SelectItem(i, String.valueOf(i))).collect(Collectors.toList());
```

**Comment by ern** [comment]
Regression it looks like from SAK-42814

**Comment by jonespm** [comment]
Yeah looks like the original was 
`		for (int i =1; i <= numberOfSlots;i++) slots.add(new SelectItem(i, i+""));`

Before this got changed to a `IntStream.range` and a 0 index. Not sure why it was 1 in the first place but this refactor did break it.

**Comment by bjones86** [comment]
Ah okay cool, good to know.

**Pull request merged**